diff_tool/diff_tool.py

```diff
@@ -148,7 +153,7 @@ def _display_in_console(stats, diffs):
 @click.option("--console", is_flag=True, help="Print results in console")
 def main(routes_file, console):
     session = api_stats.get_session_with_role(
-        role_arn="arn:aws:iam::756629837203:role/catalogue-ci"
+        role_arn="arn:aws:iam::760097843905:role/platform-ci"
```

Comment:
This loss of separation is a bit of a shame!

Reply:
Yeah, it'd be nice to get everything in the catalogue account. One day…

In the meantime, I'm tempted to go for a much broader separation here. Having the diff tool know where where the API is looking is a bit weird – it's odd that it's coupled to the implementation of the API, even though the whole point of it is that it's testing the public-facing behaviour!

I'm tempted to add an undocumented API endpoint `/_workTypes` that returns the aggregation we want here, so we know we're always looking at the same index as the API – and then the diff tool doesn't need to know any secrets, because it can get everything it needs by querying HTTP endpoints.